connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsWithCustomForwardingAdminIntegrationTest.java

```diff
@@ -0,0 +1,596 @@
+    private static final int NUM_WORKERS = 1;
+    protected Map<String, String> mm2Props = new HashMap<>();
+    protected MirrorMakerConfig mm2Config;
+    protected EmbeddedConnectCluster primary;
+    protected EmbeddedConnectCluster backup;
+    protected Exit.Procedure exitProcedure;
+    protected Properties primaryBrokerProps = new Properties();
+    protected Properties backupBrokerProps = new Properties();
+    protected Map<String, String> primaryWorkerProps = new HashMap<>();
+    protected Map<String, String> backupWorkerProps = new HashMap<>();
+    private volatile boolean shuttingDown;
+    private Exit.Procedure haltProcedure;
+
+    /*
+     * enable ACL on brokers.
+     */
+    protected static void enableAclAuthorizer(Properties brokerProps) {
+        brokerProps.put("authorizer.class.name", "kafka.security.authorizer.AclAuthorizer");
+        brokerProps.put("sasl.enabled.mechanisms", "PLAIN");
+        brokerProps.put("sasl.mechanism.inter.broker.protocol", "PLAIN");
+        brokerProps.put("security.inter.broker.protocol", "SASL_PLAINTEXT");
+        brokerProps.put("listeners", "SASL_PLAINTEXT://localhost:0");
+        brokerProps.put("listener.name.sasl_plaintext.plain.sasl.jaas.config",
+                "org.apache.kafka.common.security.plain.PlainLoginModule required "
+                        + "username=\"super\" "
+                        + "password=\"super_pwd\" "
+                        + "user_connector=\"connector_pwd\" "
+                        + "user_super=\"super_pwd\";");
+        brokerProps.put("super.users", "User:super");
+    }
+
+    /*
+     * return superUser auth config.
+     */
+    protected static Map<String, String> superUserConfig(){
+        Map<String, String> superUserClientConfig = new HashMap<>();
+        superUserClientConfig.put("sasl.mechanism", "PLAIN");
+        superUserClientConfig.put("security.protocol", "SASL_PLAINTEXT");
+        superUserClientConfig.put("sasl.jaas.config",
+                "org.apache.kafka.common.security.plain.PlainLoginModule required "
+                        + "username=\"super\" "
+                        + "password=\"super_pwd\";");
+        return superUserClientConfig;
+    }
+    protected static Map<String, String> connectorUserConfig(){
+        Map<String, String> superUserClientConfig = new HashMap<>();
+        superUserClientConfig.put("sasl.mechanism", "PLAIN");
+        superUserClientConfig.put("security.protocol", "SASL_PLAINTEXT");
+        superUserClientConfig.put("sasl.jaas.config",
+                "org.apache.kafka.common.security.plain.PlainLoginModule required "
+                        + "username=\"connector\" "
+                        + "password=\"connector_pwd\";");
+        return superUserClientConfig;
+    }
+
+
+    /*
+     * launch the connectors on kafka connect cluster and check if they are running
+     */
+    protected static void waitUntilMirrorMakerIsRunning(EmbeddedConnectCluster connectCluster,
```

Comment:
I wonder if we should call the static method in `MirrorConnectorsIntegrationBaseTest` rather than having a copy here. Same for the methods below

Reply:
Good shout, I also used the static method for `MirrorConnectorsIntegrationBaseTest.basicMM2Config`